vpc/variables.tf

```diff
@@ -44,3 +44,48 @@ variable "use_nat_gateway" {
 }
 
 variable "environment" {}
+
+variable "private_nacl_rules" {
+  type = set(object({
+    rule_no    = number
+    cidr_block = string
+    action     = string
+    from_port  = number
+    to_port    = number
+    egress     = bool
+  }))
+  default = []
+}
+
+variable "public_nacl_rules" {
+  type = set(object({
+    rule_no    = number
+    cidr_block = string
+    action     = string
+    from_port  = number
+    to_port    = number
+    egress     = bool
+  }))
+  default = []
+}
+
+variable "create_s3_gateway_endpoint" {
+  default = false
+}
+
+variable "create_dynamo_gateway_endpoint" {
+  type    = bool
+  default = false
+}
```

Comment:
As these are free should we just create these by default?

Reply:
Yeah why not. The only reason I didn't was because maybe not everyone would need them both but I think everyone is using S3 and Dynamo now.